### DISCUSSION TITLE
Crystals should have a way to "safely" destroy them

### DISCUSSION THREAD

**Discussion by mrjpjr** [FakeFishGames/Barotrauma#5073] [Ideas and suggestions]
The crystals in the Crystal Caves can only be destroyed by coming into contact with it with your character currently.

There should be a way to safely destroy them, especially because minerals/plants sometimes spawn ON them.

Maybe the same destruction mechanics as cave ice can be used?

**Comment by HeatedWafflez** [comment]
haha crystal go boom

we're gonna having a mining laser at some point in the game, regalis was working on one and even revealed it. this is probably the solution to your issue
